src/table/table.scss

```diff
@@ -122,265 +251,316 @@
+        margin-left: 0;
+        margin-right: 0;
         @include themed {
-          background-color: t(iui-color-background-4);
+          border-top: 1px solid t(iui-color-foreground-primary);
         }
+      }
 
-        > .iui-sort,
-        > .iui-sort > .iui-icon-wrapper {
-          visibility: visible;
+      // Added by Jon, not fully tested
+      + .iui-expanded-content {
+        @include themed {
+          border-color: t(iui-color-foreground-primary);
         }
       }
     }
 
-    &.iui-tables-row-active {
-      @include themed {
-        border: solid 1px t(iui-color-foreground-primary);
-        background: rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-5));
-      }
+    &:last-child {
+      padding-bottom: $iui-xxs;
+      border-bottom: none;
     }
+    // #endregion Selection
 
-    &.iui-tables-row-new {
+    // #region Statuses & states
+    &.iui-new {
       font-weight: $iui-font-weight-semibold;
-
-      > .iui-tables-filename {
-        position: relative;
-      }
-
-      > .iui-tables-filename::before {
-        @include themed {
-          color: t(iui-color-foreground-positive);
-        }
+      
+      > .iui-main-column::before {
         content: '•';
         position: absolute;
         left: 0;
         font-size: $iui-font-size-title;
+        @include themed {
+          color: t(iui-color-foreground-positive);
+        }
       }
     }
 
-    &.iui-tables-row-disabled {
+    &.iui-disabled,
+    &.iui-disabled + .iui-expanded-content {
+      font-style: italic;
+      cursor: not-allowed;
       @include themed {
         color: t(iui-text-color-muted);
       }
-      font-style: italic;
-      cursor: not-allowed;
 
-      &:hover {
-        @include themed {
-          background-color: t(iui-color-background-1);
-        }
+      &:hover > .iui-slot > .iui-more-options {
+        visibility: hidden;
+      }
 
-        > .iui-tables-slot > .iui-tables-menu {
-          visibility: hidden;
-        }
+      img,
+      svg:not(.iui-radial),
+      .iui-user-icon {
+        filter: $iui-icons-color-multicolor-disabled;
+      }
+    }
+
+    &.iui-positive {
+      @include iui-table-row-status;
+    }
+    
+    &.iui-warning {
+      @include iui-table-row-status($status:warning);
+    }
+
+    &.iui-negative {
+      @include iui-table-row-status($status:negative);
+    }
+
+    &.iui-expanded {
+      .iui-expander > .iui-icon {
+        transform: rotate(90deg);
       }
 
-      > .iui-tables-cell {
-        > img,
-        > svg,
-        > .iui-user-icon,
-        > .iui-user-icon > img {
-          filter: $iui-icons-color-multicolor-disabled;
+      + .iui-expanded-content {
+        @include themed {
+          border-top: solid 1px t(iui-color-background-1);
         }
       }
     }
+    // #endregion Selection
   }
 
-  .iui-tables-message-container {
+  // Empty & loading states
+  &.iui-message {
+    text-align: center;
+    height: 100%;
```

Comment:
again, not sure its needed..

Reply:
If I'm understanding it right, it's about Empty State ? 
I thing an empty state component should replace this iui-message